README.md

```diff
@@ -1,57 +1,215 @@
+
+   ```powershell
+   az storage account create \
+    --resource-group $ACI_PERS_RESOURCE_GROUP \
+    --name $ACI_PERS_STORAGE_ACCOUNT_NAME \
+    --location $ACI_PERS_LOCATION \
+    --sku Standard_LRS
+   ```
+
+5. Execute the commands below to create the file share.
+
+   ```powershell
+   export AZURE_STORAGE_CONNECTION_STRING=`az storage account show-connection-string --resource-group $ACI_PERS_RESOURCE_GROUP --name $ACI_PERS_STORAGE_ACCOUNT_NAME --output tsv`
+   az storage share create -n $ACI_PERS_SHARE_NAME
+   ```
+
+6. Execute the commands below to shows the storage account you created.
+
+   ```powershell
+   echo $ACI_PERS_STORAGE_ACCOUNT_NAME
+   ```
+   ![](images/deploy-02.png)
+
+## Deploy Azure Components
+
+1. Click this button to navigate to the Azure Portal deployment page.
+
+   [![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Faci-extract-load-transform-sample%2Fmaster%2Fazuredeploy.json)
+
+2. Choose the resource group created in the previous steps.
+
+3. Fill in the values on the deployment page.
+   * **Storage Account Name**: the name of the storage account created in the previous steps
+   * **Storage Share Name**: the name of the file share created in the previous steps, which is `acishare` by default
+   * **Administrator Login**:  the user name of the Postgres database
+   * **Administrator Login Password**: the password of the Postgres database, it must meet the complexity requirements, e.g. `postgres1!`
+   * **Extracting Container Image**: the Docker image you built for the extracting container
+   * **Transforming Container Image**: the docker image you built for the transforming container
+   * **Loading Container Image**: the docker image you built for the loading container
+   * **Rendering Container Image**: the docker image you built for the rendering container
+   * Check **I agree to the terms and conditions stated above**.
+
+   ![](images/deploy-03.png)
+
+4. Click **Purchase**
+
+## Check the Demo
+
+### View Demo in Web
+
+1. Open the resource group you just created.
+
+   ![](images/deploy-04.png)
+
+2. Click the container group **MS-ACIAKS-ETLContainerGroups**, notice the state of the first 3 containers is **Terminated**, while the last one is always **Running** to serve the Word Cloud.
+
+   ![](images/deploy-05.png)
+
+3. Copy the **IP address** from the container group blade, and open it in the browser to check the Word Cloud.
+
+   ![](images/deploy-06.png)
+
+### Create Index and Query Data in Azure Shell
+
+> **Note:** The following commands are running on Azure Portal.
+
+1. Open the resource group you just created.
+
+   ![](images/deploy-04.png)
+
+2. Click the Postgres database, remember the **Server name** and **Server admin login name**.
+
+   ![](images/deploy-07.png)
+
+3. Open the **Cloud Shell** on the top bar.
```

Comment:
Specify if you're using Bash or PowerShell. There isn't parity between the two, from your screenshot you're using Bash so let's specifically denote that here.

Reply:
updated